### PR TITLE
Hl 11 address, add versioning, refactor to cleaner architecture

### LINKED ISSUE #10: Add example Go tests from old project
## Desired Use Case

_Describe what this **should** implement._

## Actual/Current Behavior

_Describe what is currently happening._

## Steps to Implement This Feature

  1. 
  2. 
  3. 

## Pre-Testing TODOs

_What needs to be done before testing._

  1. Change DB values in x collection to y.

#### If you are not a member of this project, _skip this step_

_How do the users test this change?_

  1. Navigate to x screen
  2. Click on y button
  3. Observe z result

## Learning

_Describe the research stage._

_Links to blog posts, patterns, libraries or addons that could be used to solve this problem._

### PR DESCRIPTION
**Pull request by B12Howard**
## Changes

Sorry for the omnibus PR. Next time I won't cram so much stuff into it.

_List Changes Introduced by this PR._

  1. Create Repository and Service layer to follow a more clean architecture; move the models to their own directory
  2. Add versioning  constant that should be upped every time we make a PR or deploy
  3. Add unit testing to create_member 
  4. Update readme to have install PostGIS and how to run unit tests
  5. Update Address entity in migration to have apt number
 
## Purpose

_Describe the problem or feature._

## Approach

_How does this change address the problem?_

## Learning

_Describe the research stage._

_Links to blog posts, patterns, libraries or addons used to solve this problem._

Closes #11
Closes #10 
Closes #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,7 +106,8 @@ For database migrations this project uses [https://github.com/pressly/goose]
 1. Set up Firebase project and enable Authentication
 2. Configure Google Cloud Storage
 3. Set up PostgreSQL database
-4. Configure environment variables
+4. Install PostGIS PostgreSQL extension
+5. Configure environment variables
```
Comment (ShreyamPatel22) [inline README.md:110]: Looks good! Thanks for adding the PostGIS setup and test instructions

File: README.md [hunk 2]
```diff
@@ -126,3 +127,6 @@ The client application is part of the Kithli project repository.
 
 
 ![Kithli Architecture](https://user-images.githubusercontent.com/39282569/196551643-9d64515f-128e-4c8c-af39-071ce5d43226.png)
+
+ ### Running Tests
+ To run unit tests use `go test YOURDIRECTORY`. Example: Tetst are in the service directory. If I wanted to run all of them I would run `go test ./services/... -v`
```
Comment (ShreyamPatel22) [inline README.md:132]: There is a typo. Tetst should be Tests

Comment (B12Howard) [reply README.md]: Fixed

**Comment by B12Howard** [comment]
> The PR has new address fields, provides an initial integration test, and adds versioning infrastructure.
> 
> Improvements: I think adding more test cases for negative would give more accuracy. Use the actual exported HTTP handler in tests for full stack coverage.
> 
> No critical bugs found. Nice work!

You are right about adding negative cases rather than just the happy path. Could you explain using the actual exported http handler in tests? @ShreyamPatel22
